tasks/make.js

```diff
@@ -48,7 +48,10 @@ module.exports = function(grunt) {
   // source directory to catch make files included from the primary one.
```

Comment:
@grayside This comment suggests that the expansive glob pattern is meant to ensure that changes to included make files, especially with the "newer" operation. Will the change in this PR conflict with this purpose?